src/lambdaisland/embedkit.clj

```diff
@@ -95,7 +95,7 @@
     (:secret-key this)))
 
 (extend-protocol IConnection
-  clojure.lang.IAtom
+  clojure.lang.Atom
```

Comment:
I am not sure if I change it correctly. However, I read the `extend-protocol` doc, it should extend the protocol on a Type, right?